src/symmetricDifferenceWith.js

```diff
@@ -0,0 +1,29 @@
+var _curry3 = require('./internal/_curry3');
+var differenceWith = require('./differenceWith');
+var unionWith = require('./unionWith');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or
+ * second list, but not both.  Duplication is determined according to the value
+ * returned by applying the supplied predicate to two list elements.
+ *
+ * @func
+ * @memberOf R
+ * @category Relation
+ * @sig (a,a -> Boolean) -> [a] -> [a] -> [a]
```

Comment:
I'd prefer to write `a -> a -> Boolean`. As written, it's unclear whether the `,` or the `->` binds more tightly.